### PR TITLE
test: Use DSROracle in invariant tests (SC-484)

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: test/invariant/handlers/TimeBasedRateHandler.sol [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { DSRAuthOracle } from "lib/xchain-dsr-oracle/src/DSRAuthOracle.sol";
+import { IDSROracle }    from "lib/xchain-dsr-oracle/src/interfaces/IDSROracle.sol";
+
+contract TimeBasedRateHandler is StdCheats, StdUtils {
+
+    uint256 public dsr;
+    uint256 public chi;
+    uint256 public rho;
+
+    uint256 constant ONE_HUNDRED_PCT_APY_DSR = 1.00000002197955315123915302e27;
```
Comment (hexonaut) [inline test/invariant/handlers/TimeBasedRateHandler.sol:16]: Should be `1.000000021979553151239153027e27`.

File: test/invariant/handlers/TimeBasedRateHandler.sol [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    constructor(DSRAuthOracle dsrOracle_) {
+        dsrOracle = dsrOracle_;
+    }
+
+    // This acts as a receiver on an L2.
+    // Note that the chi value is not derived from previous values, this is to test if
+    // PSM will work as expected with different chi values.
+    function setPotData(uint256 newDsr, uint256 newChi, uint256 newRho) external {
+        dsr = _bound(newDsr, 1e27, ONE_HUNDRED_PCT_APY_DSR);
+        chi = _bound(newChi, chi,  1e27);
```
Comment (hexonaut) [inline test/invariant/handlers/TimeBasedRateHandler.sol:31]: `chi` is a number that is always greater than `1e27`. This should be modified to reflect that.

**Review by hexonaut — CHANGES_REQUESTED** [review]
Two issues, but lgtm other than that.

File: test/invariant/handlers/TimeBasedRateHandler.sol (hunk 1 above)
Comment (lucas-manuel) [reply test/invariant/handlers/TimeBasedRateHandler.sol]: Updated

**Review by hexonaut — APPROVED** [review]

**Pull request merged**